feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -0,0 +1,149 @@
+# RT-5.7 Aggregate Not Viable All
+
+## Summary
```

Comment:
@rszarecki can you link your issue here (as a TODO) here? (the one that's tracking test implementation?) This is for the tracking purpose.